### PR TITLE
task: add page

### PR DESCRIPTION
**Pull request by abactel**

### CODE REVIEW

File: pages/common/task.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Add new task:
+
+`task add {{thing_to_do}}`
+
+- List tasks:
+
+`task list`
+
+- Complete task:
```
Comment (waldyrious) [inline pages/common/task.md]: This is a nitpick, but "mark task as completed" would be more precise :)

**Comment by waldyrious** [comment]
@agnivade this should be ready to merge -- please take a look and do so if it looks good.

File: pages/common/task.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`task add {{thing_to_do}}`
+
+- List tasks:
+
+`task list`
+
+- Mark task as completed:
+
+`task {{task_ID}} done`
```
Comment (agnivade) [inline pages/common/task.md]: please convert ID to lowercase.

File: pages/common/task.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Modify task:
+
+`task {{task_ID}} modify {{new_thing_to_do}}`
```
Comment (agnivade) [inline pages/common/task.md]: same as above.

File: pages/common/task.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Delete task:
+
+`task {{task_ID}} delete`
```
Comment (agnivade) [inline pages/common/task.md]: same as above

**Pull request merged**